### DISCUSSION TITLE
tw-elements components lose their behavior when submitting the form

### DISCUSSION THREAD

**Discussion by guglieelmor** [livewire/livewire#7932] [1. Help]
Hey guys,

I would like to ask for help regarding a problem I'm experiencing, I'm using https://tw-elements.com/ and its components together with livewire, the input components completely lose their behavior and lose css definitions when I'm using wire:submit.prevent="create", does anyone have any idea what to do to use this wire:submit.prevent="create" structure without the tw elements components losing their functions when submitting the form?

**Comment by danie-ramdhani** [comment]
using `wire:ignore` maybe works? https://livewire.laravel.com/docs/wire-ignore

in livewire v3, you don't need `prevent` modifier. just `wire:submit="..."` needed. it prevent default action by default

**Reply by guglieelmor** [reply]
yes, it actually worked, thank you very much!!
